src/main/kotlin/com/hibob/academy/employeeFeedback/Service/FeedbackUpdateor.kt

```diff
@@ -0,0 +1,17 @@
+package com.hibob.academy.employeeFeedback.Service
+
+
+import com.hibob.academy.employeeFeedback.dao.*
+import com.hibob.academy.employeeFeedback.dao.FeedbackDao
+import jakarta.ws.rs.NotFoundException
+
+class FeedbackUpdateor(private val feedbackDao: FeedbackDao) {
+
+    fun updateStatus(companyId: Long, status: Status, feedbackId: Long): Boolean =
```

Comment:
Who is allow to update feedback status? where is the role validation?

Reply:
did it in annotation.